### DISCUSSION TITLE
github.com/login/oauth/access_token wrong certificate

### DISCUSSION THREAD

**Discussion by manuelbuil** [community/community#39745] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

We are trying to install an app for CI runs called Drone to run our CI in GitHub. We registered the OAuth application setting an `Authorization callback URL`. This url gets called correctly and then following happens:
```
{"fields.time":"2022-11-22T12:00:48Z","latency":116204,"level":"debug","method":"GET","msg":"","remote":"10.42.0.8:47342","request":"/login","request-id":"2Hty0927i1Un5kWuk5GJaftCckl","time":"2022-11-22T12:00:48Z"}
{"level":"error","msg":"oauth: cannot exchange code: fa7ed1be06add056b613: Post \"https://github.com/login/oauth/access_token\": x509: certificate is valid for cloudfront.net, *.cloudfront.net, not github.com","time":"2022-11-22T12:00:49Z"}
{"level":"debug","msg":"cannot authenticate user: Post \"https://github.com/login/oauth/access_token\": x509: certificate is valid for cloudfront.net, *.cloudfront.net, not github.com","time":"2022-11-22T12:00:49Z"}
{"fields.time":"2022-11-22T12:00:49Z","latency":19361232,"level":"debug","method":"GET","msg":"","remote":"10.42.0.8:47342","request":"/login?code=fa7ed1be06add056b613\u0026state=7a3ba6f606f665a6","request-id":"2Hty0L7UgmJIvIbHdSbPE1vKOgz","time":"2022-11-22T12:00:49Z"}
```
You can see the `,"request":"/login",`, which is the callback URL happening and then you can observe that it can't contact github anymore because:
```
https://github.com/login/oauth/access_token\": x509: certificate is valid for cloudfront.net, *.cloudfront.net, not github.com
```
Does anybody understand what is going on? thanks!

**Comment by ossanna16** [comment]
Hi there @manuelbuil and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by manuelbuil** [comment, marked as answer]
We found the problem. For some reason systemd-resolved adds the example search domain `DOMAINS` and `github.com.DOMAINS` exists. That's why we were getting a certificate problem
